packages/frontend-core/src/components/grid/stores/index.ts

```diff
@@ -44,20 +45,36 @@ const DependencyOrderedStores = [
   Users,
   Menu,
   Pagination,
-  Config,
+  Config as any,
```

Comment:
This needs to be typed as any for now, as we have some conflicts with the outputting types. It will be refactored when all the grid stores are typed